plotly_resampler/aggregation/aggregators.py

```diff
@@ -114,62 +101,55 @@ class MinMaxOverlapAggregator(AbstractSeriesAggregator):
 
     """
 
-    def __init__(self, interleave_gaps: bool = True, nan_position="end"):
+    def __init__(self, interleave_gaps: bool = True):
         """
         Parameters
         ----------
         interleave_gaps: bool, optional
```

Comment:
Types

Reply:
I'm not really sure what you are referring to? 
- are you referring to the return type of the constructor?
- or is smth wrong with how we type hint `interleave_gaps`

Reply:
nvm, just got it - `interleave_gaps` should be optional in the signature :upside_down_face:

Reply:
It's described as optional, but instead it has a default value. Is it Optional[bool] or bool with a default value? - the type listed in the documentation isn't clear on that.

Same point with the cases where I've stated 'types'.

Reply:
Thx for the clarification. 

I looked further into type hinting guidelines:
- As per [typing](https://docs.python.org/3/library/typing.html#typing.Optional) docs, it should not have `Optional` type hint.
- As per [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), you can add the default value in the parameter documentation :arrow_down: 
![image](https://user-images.githubusercontent.com/18898740/229363678-c3503242-73c0-4206-b99e-dd3baf13ce57.png)

Imo, `bool, default: True` is more clear than `bool, optional` - especially since the default value is used (creating more separation between type hings of optional types with None as default)